### PR TITLE
Added Two Google Internships (STEP & SWE Intern)

### PR DESCRIPTION
**Pull request by tohmze**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,51 +17,53 @@ To contribute:
...
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------- | --------------------- | ---------------------------------------------------------------------------------------------- | ----------- |
+| [Google](https://www.google.com/about/careers/applications/jobs/results/88626154719584966-step-intern-secondyear-student-summer-2024?src=Online%2FGoogle%20Website%2FByF&utm_source=Online%20&utm_medium=careers_site%20&utm_campaign=ByF&company=Fitbit&company=Google&company=YouTube&distance=50&employment_type=INTERN&location=Canada) | Waterloo, Montreal, Toronto             | Unknown               | STEP Intern, Second-Year Student                                                               | Oct 2nd     |
```
Comment (kanjurer) [inline README.md:22]: Maybe shorten the URL to just `https://www.google.com/about/careers/applications/jobs/results/88626154719584966-step-intern-secondyear-student-summer-2024`

File: README.md [hunk 2]
```diff
@@ -17,51 +17,53 @@ To contribute:
...
+| [Google](https://www.google.com/about/careers/applications/jobs/results/100145136748372678-software-developer-intern-bs-summer-2024?src=Online%2FGoogle%20Website%2FByF&utm_source=Online%20&utm_medium=careers_site%20&utm_campaign=ByF&company=Fitbit&company=Google&company=YouTube&distance=50&employment_type=INTERN&location=Canada)  | Waterloo, Montreal, Toronto             | Unknown               | Software Developer Intern, BS                                                                  | Oct 2nd     |
```
Comment (kanjurer) [inline README.md:23]: Same here

**Review by kanjurer — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tohmze) [reply README.md]: Thanks, done!

**Comment by jenndryden** [comment]
sweet, thanks so much 🤩

**Pull request merged**